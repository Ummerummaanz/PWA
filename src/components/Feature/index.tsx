import React from 'react';

const Feature: React.FC = () => (
  <div>{`#__FEATURE_NAME__#`}</div>
);

export default Feature;
