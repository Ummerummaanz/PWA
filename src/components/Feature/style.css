.feature-header-toolbar__root {
  display: flex;
  align-items: center;
}
