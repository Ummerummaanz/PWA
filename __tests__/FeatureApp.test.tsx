import * as React from 'react';
import { render } from '@testing-library/react';
import Feature from '../src/components/Feature';

describe('Feature', () => {
  it('should match snapshot', () => {
    const { container } = render(<Feature />);
    expect(container).toMatchSnapshot();
  });
});
