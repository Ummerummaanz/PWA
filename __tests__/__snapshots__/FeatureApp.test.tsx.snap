// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature should match snapshot 1`] = `
<div>
  <div>
    #__FEATURE_NAME__#
  </div>
</div>
`;
